src/services/read-from-disk.service.js

```diff
@@ -65,8 +65,8 @@ export function loadGuppyProjects(projectPathsInput: Array<string>) {
   // emulate a clean slate, but might also be useful for users who want to
   // create projects outside of Guppy but have them managed internally)
   try {
-    readdirSync(defaultParentPath).forEach(f => {
-      const projectPath = path.join(defaultParentPath, f);
+    readdirSync(getDefaultProjectHomePath()).forEach(f => {
+      const projectPath = path.join(getDefaultProjectHomePath(), f);
       const isDirectory = statSync(projectPath).isDirectory();
 
       if (isDirectory && !projectPaths.includes(projectPath)) {
```

Comment:
Right, so I actually don't want to do this stuff anymore.

I'll be working today on a fix that will remove this check. It doesn't make as much sense in a world where the home path is changeable, plus it makes some things a lot more complicated. So you can remove this whole block (lines 61-79)